docs/building-your-quest/actions/github-add-secret.md

```diff
@@ -0,0 +1,40 @@
+---
+title: github_add_secret
+parent: Actions
+grand_parent: Building Your Quest
+nav_order: 13
+---
+
+# github_add_secret
+
+### Description
+
+**Type**: Action
+
+Adds a secret to the user's repository.
+
+## Params
+
+- **secretName:** The name of the secret to add.
+- **secretValue:** The value of the secret.
+- **secretType** The type of secret to add. Can be one of `actions` or `codespaces`
```

Comment:
do we want to set it default to actions?

Reply:
No, rather it to be explicit to avoid confusion (It's also required on the schema)